Packs/Carbon_Black_Enterprise_Response/Integrations/CarbonBlackResponseV2/CarbonBlackResponseV2.py

```diff
@@ -861,6 +861,15 @@ def fetch_incidents(client: Client, max_results: int, last_run: dict, first_fetc
         if not alert_id or not alert_name:
             demisto.debug(f'{INTEGRATION_NAME} - Alert details are missing. {str(alert)}')
 
+        if ioc_attr := alert.get('ioc_attr'):
+            try:
+                alert['ioc_attr'] = json.loads(ioc_attr)
```

Comment:
Isn't it returned as a dictionary from the response?

Reply:
![image](https://github.com/demisto/content/assets/49711791/4562ae56-5bf7-4b69-8901-3763e2cac188)
```
        if ioc_attr := alert.get('ioc_attr'):
            demisto.debug(f"DEV_TYPE {type(ioc_attr)}") 
            demisto.debug(f"DEV_VALUE {ioc_attr}") 
            try:
                alert['ioc_attr'] = json.loads(ioc_attr)
                highlights = alert['ioc_attr'].get('highlights', {})
                for i, attribute in enumerate(highlights):
                    highlights[i] = attribute.replace("PREPREPRE", "").replace("POSTPOSTPOST", "")
            except Exception:
                pass
```

For some reason response is a string, so I had to json.load it